### ISSUE TITLE
FuzzAPI Installation Issue

### ISSUE THREAD

**Issue by thanveerk** [bundler/bundler#6033]
```
gem 'pg'
gem 'redis'
gem 'sinatra', :require => false
gem 'devise'
gem 'rollbar'
gem 'devise-bootstrap-views'

# Use ActiveModel has_secure_password
# gem 'bcrypt', '~> 3.1.7'

# Use Unicorn as the app server
# gem 'unicorn'

# Use Capistrano for deployment
# gem 'capistrano-rails', group: :development

group :development, :test do
  gem 'sqlite3'
  # Call 'byebug' anywhere in the code to stop execution and get a debugger cons
ole
  gem 'byebug'
end

group :development do
  # Access an IRB console on exception pages or by using <%= console %> in views

  gem 'web-console', '~> 2.0'

  # Spring speeds up development by keeping your application running in the back
ground. Read more: https://github.com/rails/spring
  gem 'spring'
end
```

### ../Gemfile.lock

```
GIT
  remote: https://github.com/lalithr95/API-fuzzer.git
  revision: 26b60e42caa460d9c6f456f152de0f03bab4111f
  specs:
    API_Fuzzer (0.1.0)
      activesupport
      http (~> 2.0)
      rails (>= 4.2)

GEM
  remote: https://rubygems.org/
  remote: https://rails-assets.org/
  specs:
    actionmailer (4.2.6)
      actionpack (= 4.2.6)
      actionview (= 4.2.6)
      activejob (= 4.2.6)
      mail (~> 2.5, >= 2.5.4)
      rails-dom-testing (~> 1.0, >= 1.0.5)
    actionpack (4.2.6)
      actionview (= 4.2.6)
      activesupport (= 4.2.6)
      rack (~> 1.6)
      rack-test (~> 0.6.2)
      rails-dom-testing (~> 1.0, >= 1.0.5)
      rails-html-sanitizer (~> 1.0, >= 1.0.2)
    actionview (4.2.6)
      activesupport (= 4.2.6)
      builder (~> 3.1)
      erubis (~> 2.7.0)
      rails-dom-testing (~> 1.0, >= 1.0.5)
      rails-html-sanitizer (~> 1.0, >= 1.0.2)
    activejob (4.2.6)
      activesupport (= 4.2.6)
      globalid (>= 0.3.0)
    activemodel (4.2.6)
      activesupport (= 4.2.6)
      builder (~> 3.1)
    activerecord (4.2.6)
      activemodel (= 4.2.6)
      activesupport (= 4.2.6)
      arel (~> 6.0)
    activesupport (4.2.6)
      i18n (~> 0.7)
      json (~> 1.7, >= 1.7.7)
      minitest (~> 5.1)
      thread_safe (~> 0.3, >= 0.3.4)
      tzinfo (~> 1.1)
    addressable (2.4.0)
    arel (6.0.3)
    autoprefixer-rails (6.4.1)
      execjs
    bcrypt (3.1.11)
    binding_of_caller (0.7.2)
      debug_inspector (>= 0.0.1)
    bootstrap (4.0.0.alpha3.1)
      autoprefixer-rails (>= 6.0.3)
      sass (>= 3.4.19)
    builder (3.2.2)
    byebug (9.0.5)
    coffee-rails (4.1.1)
      coffee-script (>= 2.2.0)
      railties (>= 4.0.0, < 5.1.x)
    coffee-script (2.4.1)
      coffee-script-source
      execjs
    coffee-script-source (1.10.0)
    concurrent-ruby (1.0.2)
    connection_pool (2.2.0)
    debug_inspector (0.0.2)
    devise (4.2.0)
      bcrypt (~> 3.0)
      orm_adapter (~> 0.1)
      railties (>= 4.1.0, < 5.1)
      responders
      warden (~> 1.2.3)
    devise-bootstrap-views (0.0.9)
    domain_name (0.5.20160826)
      unf (>= 0.0.5, < 1.0.0)
    erubis (2.7.0)
    execjs (2.7.0)
    globalid (0.3.7)
      activesupport (>= 4.1.0)
    highcharts-rails (4.2.5)
      railties (>= 3.1)
    http (2.0.3)
      addressable (~> 2.3)
      http-cookie (~> 1.0)
      http-form_data (~> 1.0.1)
      http_parser.rb (~> 0.6.0)
    http-cookie (1.0.3)
      domain_name (~> 0.5)
    http-form_data (1.0.1)
    http_parser.rb (0.6.0)
    i18n (0.7.0)
    jbuilder (2.6.0)
      activesupport (>= 3.0.0, < 5.1)
      multi_json (~> 1.2)
    jquery-rails (4.2.1)
      rails-dom-testing (>= 1, < 3)
      railties (>= 4.2.0)
      thor (>= 0.14, < 2.0)
    json (1.8.3)
    loofah (2.0.3)
      nokogiri (>= 1.5.9)
    mail (2.6.4)
      mime-types (>= 1.16, < 4)
    mime-types (3.1)
      mime-types-data (~> 3.2015)
    mime-types-data (3.2016.0521)
    mini_portile2 (2.1.0)
    minitest (5.9.1)
    multi_json (1.12.1)
    newrelic_rpm (3.15.2.317)
    nokogiri (1.6.8.1)
      mini_portile2 (~> 2.1.0)
    orm_adapter (0.5.0)
    pg (0.18.4)
    puma (3.4.0)
    rack (1.6.4)
    rack-protection (1.5.3)
      rack
    rack-test (0.6.3)
      rack (>= 1.0)
    rails (4.2.6)
      actionmailer (= 4.2.6)
      actionpack (= 4.2.6)
      actionview (= 4.2.6)
      activejob (= 4.2.6)
      activemodel (= 4.2.6)
      activerecord (= 4.2.6)
      activesupport (= 4.2.6)
      bundler (>= 1.3.0, < 2.0)
      railties (= 4.2.6)
      sprockets-rails
    rails-assets-tether (1.3.7)
    rails-deprecated_sanitizer (1.0.3)
      activesupport (>= 4.2.0.alpha)
    rails-dom-testing (1.0.7)
      activesupport (>= 4.2.0.beta, < 5.0)
      nokogiri (~> 1.6.0)
      rails-deprecated_sanitizer (>= 1.0.1)
    rails-html-sanitizer (1.0.3)
      loofah (~> 2.0)
    railties (4.2.6)
      actionpack (= 4.2.6)
      activesupport (= 4.2.6)
      rake (>= 0.8.7)
      thor (>= 0.18.1, < 2.0)
    rake (11.3.0)
    rdoc (4.2.2)
      json (~> 1.4)
    redis (3.3.1)
    responders (2.3.0)
      railties (>= 4.2.0, < 5.1)
    rollbar (2.8.3)
      multi_json
    sass (3.4.22)
    sass-rails (5.0.6)
      railties (>= 4.0.0, < 6)
      sass (~> 3.1)
      sprockets (>= 2.8, < 4.0)
      sprockets-rails (>= 2.0, < 4.0)
      tilt (>= 1.1, < 3)
    sdoc (0.4.1)
      json (~> 1.7, >= 1.7.7)
      rdoc (~> 4.0)
    sidekiq (4.1.4)
      concurrent-ruby (~> 1.0)
      connection_pool (~> 2.2, >= 2.2.0)
      redis (~> 3.2, >= 3.2.1)
      sinatra (>= 1.4.7)
    sinatra (1.4.7)
      rack (~> 1.5)
      rack-protection (~> 1.4)
      tilt (>= 1.3, < 3)
    spring (1.7.2)
    sprockets (3.7.0)
      concurrent-ruby (~> 1.0)
      rack (> 1, < 3)
    sprockets-rails (3.2.0)
      actionpack (>= 4.0)
      activesupport (>= 4.0)
      sprockets (>= 3.0.0)
    sqlite3 (1.3.11)
    thor (0.19.1)
    thread_safe (0.3.5)
    tilt (2.0.5)
    turbolinks (5.0.1)
      turbolinks-source (~> 5)
    turbolinks-source (5.0.0)
    tzinfo (1.2.2)
      thread_safe (~> 0.1)
    uglifier (3.0.2)
      execjs (>= 0.3.0, < 3)
    unf (0.1.4)
      unf_ext
    unf_ext (0.0.7.2)
    warden (1.2.6)
      rack (>= 1.0)
    web-console (2.3.0)
      activemodel (>= 4.0)
      binding_of_caller (>= 0.7.2)
      railties (>= 4.0)
      sprockets-rails (>= 2.0, < 4.0)

PLATFORMS
  ruby

DEPENDENCIES
  API_Fuzzer!
  bootstrap (~> 4.0.0.alpha3.1)
  byebug
  coffee-rails (~> 4.1.0)
  devise
  devise-bootstrap-views
  highcharts-rails
  jbuilder (~> 2.0)
  jquery-rails
  newrelic_rpm
  pg
  puma
  rails (= 4.2.6)
  rails-assets-tether (>= 1.1.0)!
  redis
  rollbar
  sass-rails (~> 5.0)
  sdoc (~> 0.4.0)
  sidekiq
  sinatra
  spring
  sqlite3
  turbolinks
  uglifier (>= 1.3.0)
  web-console (~> 2.0)

BUNDLED WITH
   1.13.2
```

--- TEMPLATE END ---------------------------------------------------------------
-

Unfortunately, an unexpected error occurred, and Bundler cannot continue.

First, try this link to see if there are any existing issue reports for this err
or:
https://github.com/bundler/bundler/search?q=No+such+file+or+directory+-+git+clon
e+%22https+%2F%2Fgithub.com%2Flalithr95%2FAPI-fuzzer.git%22+%22C+%2FRuby24-x64%2
Flib%2Fruby%2Fgems%2F2.4.0%2Fcache%2Fbundler%2Fgit%2FAPI-fuzzer-0e162813079405b4
aacdf29e0de5008bca020ede%22+--bare+--no-hardlinks+--quiet&type=Issues

If there aren't any reports for this error yet, please create copy and paste the
 report template above into a new issue. Don't forget to anonymize any private d
ata! The new issue form is located at:
https://github.com/bundler/bundler/issues/new

C:\Fuzzapi\bin>rake db:migrate
(in C:/Fuzzapi)
https://github.com/lalithr95/API-fuzzer.git (at master@26b60e4) is not yet check
ed out. Run `bundle install` first.

C:\Fuzzapi\bin>bundle install

**Comment by colby-swandale** [comment]
Hi, it looks like part of the error template is missing, we can't really help solve your problem without the entire error report. You can follow our [ISSUES](https://github.com/bundler/bundler/blob/master/doc/contributing/ISSUES.md) guide for more information.

**Comment by thanveerk** [comment]
Hi This is the complete error report from the cmd window.

**Comment by colby-swandale** [comment]
You will need to check your terminal's settings because it's chopping off about 1/2 of the bundler error report which we need.

**Comment by thanveerk** [comment]
[fuzzapi.txt](https://github.com/bundler/bundler/files/1306335/fuzzapi.txt)

Attached file is the complete error report for the command : bundle install

**Comment by colby-swandale** [comment]
Bundler is raising an exception because it's trying to clone a git repository but git cannot be found. Do you have git installed on this machine?

**Comment by thanveerk** [comment]
Hey, thanks for the reply. 

Could you please help me with this error down here:



C:\Users\Sreekanth2\Fuzzapi\bin>bundle install
The latest bundler is 1.16.0.pre.2, but you are currently running 1.15.4.
To update, run `gem install bundler --pre`
Fetching gem metadata from https://rails-assets.org/...
Fetching version metadata from https://rails-assets.org/..
Fetching gem metadata from https://rails-assets.org/...
Fetching gem metadata from https://rubygems.org/........
Fetching version metadata from https://rails-assets.org/..
Fetching version metadata from https://rubygems.org/...
Fetching dependency metadata from https://rails-assets.org/..
Fetching dependency metadata from https://rubygems.org/..
Resolving dependencies...
Using rake 11.3.0
Using i18n 0.7.0
Fetching json 1.8.3
Installing json 1.8.3 with native extensions
Using minitest 5.9.1
Using thread_safe 0.3.5
Using addressable 2.4.0
Using unf_ext 0.0.7.2 (x64-mingw32)
Using http-form_data 1.0.1
Fetching http_parser.rb 0.6.0
Installing http_parser.rb 0.6.0 with native extensions
Using builder 3.2.2
Using erubis 2.7.0
Using mini_portile2 2.1.0
Using rack 1.6.4
Using mime-types-data 3.2016.0521
Using arel 6.0.3
Using bundler 1.15.4
Using thor 0.19.1
Using concurrent-ruby 1.0.2
Using execjs 2.7.0
Using bcrypt 3.1.11 (x64-mingw32)
Fetching debug_inspector 0.0.2
Installing debug_inspector 0.0.2 with native extensions
Using sass 3.4.22
Fetching byebug 9.0.5
Installing byebug 9.0.5 with native extensions
Using coffee-script-source 1.10.0
Using connection_pool 2.2.0
Using orm_adapter 0.5.0
Using devise-bootstrap-views 0.0.9
Using multi_json 1.12.1
Using newrelic_rpm 3.15.2.317
Using pg 0.18.4 (x64-mingw32)
Fetching puma 3.4.0
Installing puma 3.4.0 with native extensions
Using rails-assets-tether 1.3.7
Using redis 3.3.1
Using tilt 2.0.5
Using spring 1.7.2
Using sqlite3 1.3.11 (x64-mingw32)
Using turbolinks-source 5.0.0
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

current directory:
C:/Ruby24-x64/lib/ruby/gems/2.4.0/gems/json-1.8.3/ext/json/ext/generator
C:/Ruby24-x64/bin/ruby.exe -r ./siteconf20170918-4296-nrbyjs.rb extconf.rb
creating Makefile

current directory:
C:/Ruby24-x64/lib/ruby/gems/2.4.0/gems/json-1.8.3/ext/json/ext/generator
make "DESTDIR=" clean
'make' is not recognized as an internal or external command,
operable program or batch file.

current directory:
C:/Ruby24-x64/lib/ruby/gems/2.4.0/gems/json-1.8.3/ext/json/ext/generator
make "DESTDIR="
'make' is not recognized as an internal or external command,
operable program or batch file.

make failed, exit code 1

Gem files will remain installed in
C:/Ruby24-x64/lib/ruby/gems/2.4.0/gems/json-1.8.3 for inspection.
Results logged to
C:/Ruby24-x64/lib/ruby/gems/2.4.0/extensions/x64-mingw32/2.4.0/json-1.8.3/gem_ma
ke.out

An error occurred while installing json (1.8.3), and Bundler cannot continue.
Make sure that `gem install json -v '1.8.3'` succeeds before bundling.

In Gemfile:
  API_Fuzzer was resolved to 0.1.0, which depends on
    rails was resolved to 4.2.6, which depends on
      actionmailer was resolved to 4.2.6, which depends on
        actionpack was resolved to 4.2.6, which depends on
          actionview was resolved to 4.2.6, which depends on
            rails-dom-testing was resolved to 1.0.7, which depends on
              rails-deprecated_sanitizer was resolved to 1.0.3, which depends on

                activesupport was resolved to 4.2.6, which depends on
                  json

C:\Users\Sreekanth2\Fuzzapi\bin>rake db:migrate
(in C:/Users/Sreekanth2/Fuzzapi)
Could not find json-1.8.3 in any of the sources
Run `bundle install` to install missing gems.

**Comment by colby-swandale** [comment]
> 'make' is not recognized as an internal or external command,

It appears to don't have the necessary tools available to compile the `JSON` gem into your env. You will need to consult the documentation for the ruby package your using.
